process_scraped_data/process_corenlp_locations_corenlp_output.R

```diff
@@ -49,3 +49,6 @@ outfile = args[2]
 
 loc_res = read_result_files(corenlp_output_dir)
 write.table(loc_res, file=outfile, sep="\t", quote=F, row.names=F)
+
+rm(list=ls())
```

Comment:
Don't you need also this `rm(...)` just before the call to `gc()` in lines 33-35 above (not with all variables defined I guess)?

Reply:
Please, ignore these comments on R code if you think are not necessary. I don't use R much.

Reply:
So in this case I don't want to include the `rm(...)` call. In the method `get_osm_locations` a large file is being read, but should be freed from memory once the object is out of scope. However, the garbage collector wasn't being called frequently enough to clear up all the objects that are out of scope.